Add Ramp to IIIF viewer matrix

Dananji opened the pull request:
This PR proposes to add [Ramp](https://github.com/samvera-labs/iiif-react-media-player) (previously known as React IIIF Media Player) to Cookbook viewer matrix. Ramp only supports A/V manifests.

It supports the following recipes;
- [Simplest Manifest - Audio](https://iiif.io/api/cookbook/recipe/0002-mvm-audio/)
- [Simplest Manifest - Video](https://iiif.io/api/cookbook/recipe/0003-mvm-video/)
- [Begin playback at a specific point - Time-based media (start)](https://iiif.io/api/cookbook/recipe/0015-start/)
- [Providing Access to Transcript Files of A/V Content (rendering)](https://iiif.io/api/cookbook/recipe/0017-transcription-av/)
- [Table of Contents for A/V Content](https://iiif.io/api/cookbook/recipe/0026-toc-opera/)
- [Table of Contents for Multiple A/V Files on a Single Canvas (start)](https://iiif.io/api/cookbook/recipe/0064-opera-one-canvas/)
- [Table of Contents for Multiple A/V Files on Multiple Canvases](https://iiif.io/api/cookbook/recipe/0065-opera-multiple-canvases/)
- [Using Caption and Subtitle Files in Multiple Languages with Video Content](https://iiif.io/api/cookbook/recipe/0074-multiple-language-captions/)
- [Using Caption and Subtitle Files with Video Content](https://iiif.io/api/cookbook/recipe/0219-using-caption-file/)

glenrobson commented:
Thanks Dananji this is great to see. I can see this pull request is still in draft but let me know when its ready. 

The next cookbook meeting is on the 6th of April and if its ready we can go through it and merge.

glenrobson commented:
We reviewed the functionality compared to the recipes and all looked OK apart from the following one:

For the [transcript recipe](https://preview.iiif.io/cookbook/ramp-in-cookbook/recipe/0017-transcription-av/) we can't download the txt file although we can see the download button. 

Also for the other recipes we noted the [label](https://iiif.io/api/presentation/3.0/#label) and [metadata](https://iiif.io/api/presentation/3.0/#metadata) don't show even though according to the spec it is required. Do you have any plans to show the label and metadata?

Dananji commented:
Glen, I've changed the instructions for the transcript recipe clarify how to download the files in Ramp. And changed the interface in Ramp a little bit, which is now deployed in our demo site.
